### PR TITLE
chg: use bor and root chain for testing

### PR DESCRIPTION
**Pull request by jdkanani**
https://forum.openzeppelin.com/t/require-in-view-pure-functions-dont-revert-on-public-networks/1211

### CODE REVIEW

File: test/child/ChildChainManager.test.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ contract('ChildChainManager', async(accounts) => {
     })
 
     it('Token should not exist before deposit', async() => {
-      await expectRevert(contracts.child.dummyERC721.ownerOf(depositTokenId), 'ERC721: owner query for nonexistent token')
+      await expectRevert(contracts.child.dummyERC721.ownerOf(depositTokenId), 'value out of range')
```
Comment (jdkanani) [inline test/child/ChildChainManager.test.js:189]: This is regarding -- https://forum.openzeppelin.com/t/require-in-view-pure-functions-dont-revert-on-public-networks/1211

File: test/helpers/contracts.js [hunk 2]
```diff
@@ -26,28 +28,36 @@ const rootProvider = new Web3.providers.HttpProvider(rootRPC)
 const childProvider = new Web3.providers.HttpProvider(childRPC)
 
 const rootWeb3 = new Web3(rootProvider)
+rootWeb3.setNetworkType = () => {} // Truffle work around for Web3Shim
```
Comment (jdkanani) [inline test/helpers/contracts.js:31]: This is workaround for truffle's web3 wrapper.

File: test/helpers/contracts.js [hunk 3]
```diff
@@ -26,28 +28,36 @@ const rootProvider = new Web3.providers.HttpProvider(rootRPC)
...
 const childWeb3 = new Web3(childProvider)
+childWeb3.setNetworkType = () => {} // Truffle work around for Web3Shim
+
+// set web3 and provider
+const setWeb3 = (contractObj, w3) => {
+  contractObj.web3 = w3
+  contractObj.setProvider(w3.currentProvider)
```
Comment (jdkanani) [inline test/helpers/contracts.js:38]: Truffle also needs `setProvider` to change network for particular contract.

File: test/child/ChildERC721.test.js [hunk 4]
```diff
@@ -30,7 +30,7 @@ contract('ChildERC721', (accounts) => {
     })
 
     it('Token should not exist before deposit', async() => {
-      await expectRevert(contracts.dummyERC721.ownerOf(tokenId), 'ERC721: owner query for nonexistent token')
+      await expectRevert(contracts.dummyERC721.ownerOf(tokenId), 'value out of range')
```
Comment (arthcp) [inline test/child/ChildERC721.test.js:33]: So we should expect these tests to fail locally? And how did the tests pass on CI in previous PRs?

File: test/child/ChildChainManager.test.js (hunk 1 above)
Comment (arthcp) [reply test/child/ChildChainManager.test.js:189]: So we should expect these tests to fail locally? And how did the tests pass on CI in previous PRs?

Comment (jdkanani) [reply test/child/ChildChainManager.test.js:189]: @arthcp It was running on ganache instead of Bor.

**Pull request merged**